app/src/pages/inside/membersPage/membersGrid/membersGrid.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import { Grid } from 'components/main/grid';
+import { PersonalInfo } from './personalInfo';
+import { LastLogin } from './lastLogin';
+import { ProjectRole } from './projectRole';
+import { UnassignAction } from './unassignAction';
+import styles from './membersGrid.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  nameCol: { id: 'MembersGrid.nameCol', defaultMessage: 'Name / Login' },
+  lastLoginCol: { id: 'MembersGrid.lastLoginCol', defaultMessage: 'Last login' },
+  roleCol: { id: 'MembersGrid.roleCol', defaultMessage: 'Project role' },
+  actionCol: { id: 'MembersGrid.actionCol', defaultMessage: 'Action' },
+});
+
+const NameColumn = ({ className, ...rest }) => (
+  <div className={cx('name-col', className)}>
+    <PersonalInfo
+      name={rest.value.full_name}
+      login={rest.value.userId}
+      userRole={rest.value.userRole}
+    />
+  </div>
+);
+NameColumn.propTypes = {
+  className: PropTypes.string.isRequired,
+};
+
+const LastLoginColumn = ({ className, ...rest }) => (
+  <div className={cx('last-login-col', className)}>
+    <LastLogin time={rest.value.last_login} />
```

Comment:
If you do not pass rest props to child components, I think you can get value prop directly from destruction of arguments. And is there rest prop really needed?